tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+env_list =
+    py38
+    py39
+    py310
+    py311
+min_version = 4.4.4
```

Comment:
I'm not sure if this matters, but doesn't match min tox version in pyproject.toml

Reply:
Addressed this in c842011f585de25de8d6a454d93416f61a65c672 by setting identical minimum versions and adding a comment in `pyproject.toml` to indicate they should ideally be kept in sync.

Hopefully in the future tox will add native support for keeping its config in `pyproject.toml` and that the redundancy will go away as a result. This is OK as-is, though.